ui/src/components/common/InviteAcceptTile.tsx

```diff
@@ -1,12 +1,14 @@
 import React from 'react'
 
+import { MarketRole } from '@daml.js/da-marketplace/lib/Marketplace/Utils'
+
+import { indefiniteArticle as a, roleLabel } from './utils'
```

Comment:
This was there before, but just now I had to trace back for the origin of `a` because I didnt know what it meant. Possibly a name more descriptive would benefit those interpreting the code - like `formatIndefiniteArticle()`

Reply:
Good point, I'll rename that function